docs/source/install.rst

```diff
@@ -44,26 +44,27 @@ binary dependencies from ``apt`` by running:
 
 .. code-block:: bash
 
-   $ sudo apt-get install libatlas-base-dev python-dev gfortran pkg-config libfreetype6-dev
+   $ sudo apt-get install libatlas-base-dev python-dev gfortran pkg-config libfreetype6-dev hdf5
```

Comment:
Is this the right package name? `apt-get install hdf5` fails for me with `Unable to locate package hdf5`.

We've been `apt-get` installing `hdf5-tools`.

Reply:
Ah, I figured it might be wrong. I'll update.